src/components/PopupWithImage.js

```diff
@@ -11,7 +11,7 @@ export default class PopupWithImage extends Popup {
     this._cardImage.alt = cardData.name
     this._cardImageTitle.textContent = cardData.name;
     super.open();
-
+    super.setEventListeners();

```

Comment:
`    super.setEventListeners();
`

Please, delete it

If you didn't have it in the 8th sprint, you shouldn't add it now

`this.setEventListeners()` should be removed from `open` because it's a **public** method that should be called only **once** in `index.js` otherwise you add listeners again and again to the same elements. This may cause a **`memory leak`**